app/controllers/root/index.controller.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const paths = require('../../paths')
+const formatAccountPathsFor = require('../../utils/format-account-paths-for')
+
+const get = function index (req, res) {
```

Comment:
The const name is different to the function name

I think we decided
```
function get (req, res) {
...
}
```
without assigning to a constant was best to avoid this.

Reply:
Updated!